schemas/geodesyml/0.6/monumentInfo.xsd

```diff
@@ -179,7 +179,8 @@ Copyright: Scripps Orbit and Permanent Array Center (SOPAC), Commonwealth Govern
         <complexContent>
             <extension base="gml:AbstractGMLType">
                 <sequence>
-                    <element maxOccurs="unbounded" name="dataCenter" type="string"/>
+                    <element maxOccurs="unbounded" name="dataCenter" type="geo:DataCenterCodeType"/>
+	                <element minOccurs="0" maxOccurs="unbounded" name="gnssNetwork" type="geo:NetworkCodeType"/>
                     <element name="urlForMoreInformation" type="string"/>
```

Comment:
If dataCenter is a URL, I'm not sure this is needed.

Reply:
I suppose it isn't necessarily? In some examples, it is a URL, and in others it is not.